utils/Makefile

```diff
@@ -62,45 +63,45 @@ viewcover:
 # Conditional virtualenv building strategy taken from this great post
 # by Marcel Hellkamp:
 # http://blog.bottlepy.org/2012/07/16/virtualenv-and-makefiles.html
-venv: oo-installenv/bin/activate
-oo-installenv/bin/activate: test-requirements.txt
+$(VENV): $(VENV)/bin/activate
+$(VENV)/bin/activate: test-requirements.txt
 	@echo "#############################################"
 	@echo "# Creating a virtualenv"
 	@echo "#############################################"
-	test -d venv || virtualenv $(NAME)env
-	. $(NAME)env/bin/activate && pip install setuptools==17.1.1
-	. $(NAME)env/bin/activate && pip install -r test-requirements.txt
-	touch $(NAME)env/bin/activate
+	test -d $(VENV) || virtualenv $(VENV)
+	. $(VENV)/bin/activate && pip install setuptools==17.1.1
```

Comment:
I want to say that we hit some type of an issue with syntax between versions, and we pinned the specific version to know that it would work, instead of fighting with trying to get the syntax exactly right. At some point, we probably want to revisit that.